### PR TITLE
✨ Songs

### PR DESCRIPTION
**Pull request by oliverlevay**

### CODE REVIEW

File: frontend/pages/songs/index.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      <h2>{t('songs')}</h2>
+      <SongsList />
+    </>
+  );
+}
+
+export async function getStaticProps({ locale }) {
+  return {
+    props: {
+      ...(await serverSideTranslations(locale, ['common', 'news'])),
```
Comment (LucasBoberg) [inline frontend/pages/songs/index.tsx:18]: Should only have common

**Review by LucasBoberg — CHANGES_REQUESTED** [review]
I have only one comment about loading locales. But there should also probably be added a query for fetching a single song using an id. Which could be used in the /songs/:id page

**Comment by oliverlevay** [comment]
> I have only one comment about loading locales. But there should also probably be added a query for fetching a single song using an id. Which could be used in the /songs/:id page

Thank you for your review! I have implemented your requested changes.

**Review by LucasBoberg — APPROVED** [review]
LGTM! :)

**Pull request merged**